instance.tf

```
@@ -21,7 +21,7 @@ resource "aws_security_group" "public" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    cidr_blocks = ["98.56.189.252/32"]
+    cidr_blocks = ["76.137.140.36/32"]
```

Looks like your public IP changes often. Let's have it as a variable
```suggestion
    cidr_blocks = ["${var.my_public_ip}/32"]
```

---

Changed hard coded ip address to a variable